NETPROGRAM/BasicExercise/Exercise15.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NETPROGRAM.BasicExercise
+{
+    public class Exercise15
+    {
+
+        public int CountZeroFactorial(int n)
+        {
+            var count = 0;
+
+            for(int i= 1; i <= n; i++)
```

Comment:
Anh làm gì dữ vậy, nó chỉ thế này thôi.

Chạy từ 1 đến n, mỗi số ví dụ là i thì tìm số n lớn nhất sao cho i chia hết cho 5^n. 

Kết quả bài toán là tổng của các số n.

Ví dụ cho 30! Chạy từ 1 --> 30

- 1 ... 4 --> 0 vì 5^0 = 1 là số lớn nhất mà các số này chia hết
- 5 --> 1 
- 6..9 --> 0
- 10 --> 1
- 11..14 --> 0
- 15 --> 1
- 16..19 --> 0
- 20 --> 1
- 21..24 --> 2 (vì 25 chia hết cho 5^2)
- 26 .. 29 --> 0
- 30 --> 1

Vậy kết quả là 1 + 1 + 1 + 1 + 2 + 1 = 7

Có thể kiểm chứng ở đây
https://www.wolframalpha.com/input/?i=30%21

![image](https://user-images.githubusercontent.com/10450325/79059228-92a3ab00-7ca1-11ea-9b4f-8abbc1aa162f.png)

Trên đây là lời giải thủ công nhất, thật ra có 1 cách khác dễ hơn nếu anh tinh ý nhìn ra.

Reply:
done